src/components/SettingsLoader.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <div id="settings-loader">
+        <slot v-if="isLoaded" />
+
+        <div v-else :class="['modal', 'z-top', {'is-active': error}]">
+            <div class="modal-content">
+                <div class="notification is-danger">
+                    <h3 class="title">Error</h3>
+                    <h5 class="title is-5">{{error && error.name}}</h5>
+                    <p>{{error && error.message}}</p>
+                    <div v-if="error && error.solution">
+                        <br/>
+                        <h5 class="title is-5">Suggestion</h5>
+                        <p>{{error && error.solution}}</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop } from "vue-property-decorator";
+
+import R2Error from "../model/errors/R2Error";
+import GameManager from "../model/game/GameManager";
+import ManagerSettings from "../r2mm/manager/ManagerSettings";
+
+const url = "https://github.com/ebkr/r2modmanPlus/wiki/Error:-White-or-blank-game-select-screen-on-startup";
+const solution = `Resetting settings might help. For more information, see ${url}`;
+
+@Component
+export default class SettingsLoader extends Vue {
+    @Prop({required: true})
+    private logError!: (error: R2Error) => void;
+
+    error: R2Error|null = null;
+    isLoaded = false;
+
+    get defaultGame() {
+        const defaultGame = GameManager.unsetGame();
+
+        // Don't trust the non-null asserted typing of .unsetGame().
+        if (defaultGame === undefined) {
+            throw new Error("GameManager.unsetGame() returned undefined");
+        }
+
+        return defaultGame;
+    }
+
+    handleError(name: string, message: string, solution: string|null = null) {
+        this.error = new R2Error(name, message, solution);
+        this.logError(this.error);
+    }
+
+    async created() {
+        let defaultGame;
```

Comment:
Why are we using a computed property if it's only used within this block?

Reply:
I'd prefer a function over a computed property in this case, but that aside I'd think splitting code to standalone functions where possible like this makes sense

Reply:
I agree, computed property doesn't make much sense here, [extracted to a separate function](https://github.com/ebkr/r2modmanPlus/pull/911/files#diff-449d18893f0dea4f8c02461069f6129fde06e2558467115d7766953e2f3b43abR81).

Reply:
Well, seeing what that change does to the next PR in the chain, I'd flip this around and as why not use computed property (or plain method) here? Apart from the theoretical easier testing?

Reply:
I don't mind it either way around, but in the interest of pushing the changes forward, I think we should merge this version now & make more changes in the latter PR if necessary